libdnf/module/module_sack.cpp

```diff
@@ -265,6 +266,9 @@ ModuleSack::Impl::collect_data_for_modular_filtering() {
 void ModuleSack::Impl::module_filtering() {
     auto [include_NEVRAs, exclude_NEVRAs, names, src_names, reldep_name_list] = collect_data_for_modular_filtering();
```

Comment:
To avoid unnecessary warning and work we have to test first whether there are any modular data in ModuleSack (first line). If it is empty it have to return without doing anything.